Xamarin.PropertyEditing.Mac/Controls/PropertyEditorControl.cs

```diff
@@ -89,7 +91,13 @@ public void UpdateKeyViews ()
 				} while (row > 0 && ctrl == null);
 
 				if (ctrl != null) {
-					ctrl.LastKeyView.NextKeyView = FirstKeyView;
+					VariationButton vb = Superview.Subviews.OfType<VariationButton> ().FirstOrDefault ();
```

Comment:
Agreed. I'd like to do that, but think it should be done under a separate issue, assuming this works and doesn't affect performance.

Reply:
Making outside changes to better enable new scenarios don't have to be separate issues, that's just going to accumulate tech debt because they'll get lost in the priority, especially when it wouldn't have taken that long to have just done it that way in the first place.

Reply:
I'd totally focused on the 2nd. Just implemented the 1st.